specs/paths.yaml

```
@@ -25,9 +25,9 @@
     $ref: paths/users/post.yaml
   get:
     $ref: paths/users/get.yaml
-/users/{userId}/position-permission/:
+/users/{userId}/update-permission/:
   patch:
-    $ref: paths/users/_userId/position-permission/patch.yaml
+    $ref: paths/users/_userId/update-permission/patch.yaml
```

như thế này thôi em. HTTP method `patch` là đủ để nói nó là update rồi.
```suggestion
    $ref: paths/users/_userId/permission/patch.yaml
```